.github/workflows/docs-ci.yaml

```diff
@@ -82,3 +82,6 @@ jobs:
 
       - run: make ${{ inputs.make-target }}
         working-directory: ${{ inputs.docs-directory }}
+
+      - run: SPHINXOPTS="-W --keep-going" make linkcheck ${{ inputs.make-target }}
```

Comment:
_non-blocking question_ 

Is `SPHINXOPTS="-W --keep-going"` meant to override the default envvar set in the workflow? 
https://github.com/nextstrain/.github/blob/429f042333c3cbfd2587d2942d283a57d3d252cc/.github/workflows/docs-ci.yaml#L48

Reply:
> _non-blocking question_
> 
> Is `SPHINXOPTS="-W --keep-going"` meant to override the default envvar set in the workflow?
> 
> https://github.com/nextstrain/.github/blob/429f042333c3cbfd2587d2942d283a57d3d252cc/.github/workflows/docs-ci.yaml#L48

TBPH, I copied that invocation [from Tom's original note in the issue](https://github.com/nextstrain/.github/issues/29#issue-1442955952), and I didn't notice there was a default setting. Experimenting locally, `-n` doesn't seem to make a difference one way or the other, so I'll remove the env var in this step.